data.sql

```diff
@@ -41,4 +41,51 @@ INSERT INTO
         weigth_kg
     )
 VALUES
-    ('Devimon', '2017-05-12', 5, true, 11.0);
\ No newline at end of file
+    ('Devimon', '2017-05-12', 5, true, 11.0);
+
+INSERT INTO
+    animals(
+        name,
+        date_of_birth,
+        escape_attempts,
+        neutered,
+        weigth_kg
+    )
+VALUES
+    ('Charmander', '2020-02-08', 0, false, 11.0);
+
+INSERT INTO
+    animals(
+        name,
+        date_of_birth,
+        escape_attempts,
+        neutered,
+        weigth_kg
+    )
+VALUES
+    ('Plantmon', '2021-11-15', 2, true, 5.7);
+
```

Comment:
- [ ] Nice work inserting the animal **Plantmon** data into the `animals` table :clap: However, according to the project requirement the weight of the animal is `-5.7kg` but in your SQL code you're inserting  `5.7kg`. Therefore I suggest you insert the right weight (-5.7) to follow the project requirement.
```sql
INSERT INTO
    animals(
        name,
        date_of_birth,
        escape_attempts,
        neutered,
        weigth_kg
    )
VALUES
    ('Plantmon', '2021-11-15', 2, true, -5.7);
```